src/dblib.mli

```diff
@@ -182,6 +182,11 @@ val err_handler : (severity -> int -> string -> unit) -> unit
     is given the {!severity}, the number of the error (see the 400 or
     so error numbers sybdb.h, macros SYBE...) and the message. *)
 
+val msg_handler : (severity -> int -> string -> unit) -> unit
+(** [msg_handler f] installs [f] as the default message handler for
+    messages coming from the server. [f] is given the message, {!severity}, and
+    line number. *)
+
```

Comment:
Thanks for your changes.  Could you please adapt the documentation to the new order?